# MotorPos.msg

int32 servo_id          
int32[] servo_ids      
int32[] position        
int32[] speed           
int32[] acceleration    
